pkg/cli/volumes.go

```
@@ -34,7 +40,12 @@ func (a *Volume) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	apps, _ := c.AppList(cmd.Context())
```

Let's not ignore this error.